incubator/kubevirt/templates/apiserver-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kubevirt.fullname" . }}-virt-api
+  namespace: {{ .Values.kubevirt.namespace }}
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+    component: "virt-api-server"
+spec:
+  ports:
+    - port: 443
+      targetPort: 8443
+      protocol: TCP
+  selector:
+{{ include "selectors.standard" . | indent 4 }}
```

Comment:
`chart` and `heritage` should not be part of the selector.

Reply:
fixed